ckanext/announcements/helpers.py

```diff
@@ -35,3 +35,8 @@ def get_all_announcements():
     )
 
     return messages
+
+
+def get_user_env(user_dict):
```

Comment:
If this (really scary) function is only used in tests add it to the relevant test module and not the public helpers

Reply:
Updated here https://github.com/okfn/ckanext-announcements/commit/4a08aebcd23eb5bcf16a778b29fe2d51fd4e189d and finally removed the helper here https://github.com/okfn/ckanext-announcements/commit/95e68f918f99d31452b09d325e2684a6aa3308b2 @amercader